libraries/ms-common/src/stm32f0xx/gpio.c

```diff
@@ -35,11 +35,6 @@ static bool prv_are_settings_valid(const GPIOSettings *settings) {
 
 
 StatusCode gpio_init(void) {
```

Comment:
Do we need a deinit all but UART function for testing? I don't think we need this for anything else though.